src/components/o-pagination/_template.js

```diff
@@ -0,0 +1,23 @@
+import html from 'nanohtml';
+import raw from 'nanohtml/raw';
+import classnames from 'classnames';
+
+export default ({ classes, items, first, last }, childrenFragment) => html`
+  <nav class="o-pagination ${classes}">
+    ${childrenFragment}
+
+    ${(first && html`
+      <a class="o-pagination__first" href=${''} />${raw('&lt;')}</a>
+    `) || ''}
+
+    ${Array.isArray(items) && items.map(({ href, active, page }, index) => html`
```

Comment:
Think we would have a limit of max pages, right?

Or actually there are two major approaches on pagination:
- the available number of pages are known
- VS the available number of are **not** known